frontend/packages/operator-lifecycle-manager/src/plugin.tsx

```diff
@@ -263,6 +268,29 @@ const plugin: Plugin<ConsumedExtensions> = [
           .InstallPlansPage,
     },
   },
+  {
+    type: 'Dashboards/Overview/Health/Operator',
+    properties: {
+      title: 'Operators',
+      resources: [
+        {
+          kind: referenceForModel(models.ClusterServiceVersionModel),
+          isList: true,
+          prop: 'clusterServiceVersions',
+        },
+        {
+          kind: referenceForModel(models.SubscriptionModel),
+          prop: 'subscriptions',
+          isList: true,
+        },
+      ],
+      getOperatorsWithStatuses: getClusterServiceVersionsWithStatuses,
+      operatorRowLoader: () =>
+        import(
```

Comment:
Let's stay consistent in how we use dynamic imports within this specific module:

```ts
loader: async () =>
  (await import('./components/dashboard/csv-status' /* webpackChunkName: "csv-dashboard-status" */))
    .default,
```

Reply:
fixed